presto-native-execution/README.md

```diff
@@ -161,7 +166,9 @@ Run CLion:
 
 ### Run Presto Coordinator + Worker
 * Note that everything below can be done w/o using IDEs by running command line commands (not in this readme).
-* Run `HiveExternalWorkerQueryRunner` from IntelliJ and wait until it starts (`======== SERVER STARTED ========` is displayed in the log output).
+* Run ExternalWorkerQueryRunner as per your choice,
```

Comment:
There is no ExternalWorkerQueryRunner class. Maybe just use plain English words?